content/_en.html

```diff
@@ -3,75 +3,347 @@
+                    </div>
+                </div>
+
+                <div class="list__item">
+                    <div class="teaser teaser--basic">
+                        <div class="teaser__heading">
+                            <h3 class="teaser__title">Get a deeper understanding of CUE</h3>
+                        </div>
+                        <div class="teaser__buttons">{{< button href="#" label="Language guide" theme="blue" >}}</div>
+                    </div>
+                </div>
+
+                <div class="list__item list__item--wide">
+                    <div class="teaser teaser--basic">
+                        <div class="teaser__heading">
+                            <h3 class="teaser__title">Download your version of CUE</h3>
+                        </div>
+                        <div class="teaser__buttons">
+                            {{< button href="#" label="Download center" theme="yellow" >}} {{< button href="#"
+                            label="Github" theme="outline" >}}
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="section section--center section--yellow section--icon-teasers">
+    <header class="section__header">
+        <div class="section__content">
+            <h2 class="section__title">Infinite possibilities</h2>
+            <div class="section__intro">
+                <p>
+                    CUE is an open source language, with a rich set of APIs and tooling, for defining, generating, and
+                    validating all kinds of data: configuration, APIs, database schemas, code, … you name it.
+                </p>
+            </div>
+        </div>
+    </header>
+
+    <div class="section__body">
+        <div class="section__container">
+            <div class="list list--33">
+                <div class="list__item">
+                    <div class="icon-teaser">
+                        <div class="icon-teaser__media">
+                            <svg class="icon icon-teaser__icon" aria-hidden="true">
+                                <use
+                                    xmlns:xlink="http://www.w3.org/1999/xlink"
+                                    xlink:href="/img/ui.svg#icon--circle-hollow"
+                                ></use>
+                            </svg>
+                        </div>
+                        <h3 class="icon-teaser__title">Data validation</h3>
+                        <p class="icon-teaser__text">
+                            Validate text-based data files or programmatic data such as incoming RPCs or database
+                            documents.
+                        </p>
+                        <p class="link icon-teaser__readmore">
+                            <span class="link__text">Learn more</span>
+                        </p>
+                        <a class="icon-teaser__link" href="#">
```

Comment:
not sure what we decided on accessibility level but in case we want to have level A I think it needs an aria attribute or extra sr-only text. "Learn more about Data Validation" for instance. See https://www.visionaustralia.org/business-consulting/digital-access/blog/how-to-make-read-more-links-accessible.